Neos.Neos/Classes/Fusion/AbstractMenuItemsImplementation.php

```diff
@@ -175,6 +178,9 @@ protected function getNodeLevelInSite(Node $node): int
     {
```

Comment:
getNodeLevelInSite seems to be dead code now ^^, previously only used in getCurrentNodeRootline

Reply:
and removed it is